index.md

```diff
@@ -147,3 +147,13 @@ The nature of our project is such that we aren't actually utilizing large datase
 
 ### Analysis
 In terms of analysis and assessment, we have altered the original *AI for Snake Game* repository to provide us with cleaned data describing the performance of the genetic algorthm's training, however we will need to look into the tools provided by OpenAI Gym for analysis and assessment so that we may compare the two methods for training snake agents.
+
+## Discussion Outline
```

Comment:
Looks awesome overall, definitely as nice template for us to work from. The only thing I can think of adding as a section would be something explaining how we put all of our tests (RL vs. GA, A2C vs. DQN, etc.) on a level playing field and could compare them. 

Basically, just a way to hype up your sick Jupyter NB!

Reply:
I don't think this goes in Discussion Outline... I added in a sentence to methods.